package.json

```diff
@@ -21,37 +21,38 @@
   "scripts": {
     "lint": "eslint .",
     "pretest": "npm run lint",
-    "test": "nyc mocha --async-only",
-    "azure-pipelines": "nyc mocha --async-only --reporter xunit -O output=test.xunit",
-    "coveralls": "nyc report --reporter=text-lcov | coveralls"
+    "test": "nyc mocha --async-only"
   },
   "dependencies": {
-    "arr-flatten": "^1.0.1",
-    "arr-map": "^2.0.0",
-    "bach": "^1.0.0",
-    "collection-map": "^1.0.0",
-    "es6-weak-map": "^2.0.1",
-    "last-run": "^1.1.0",
-    "object.defaults": "^1.0.0",
-    "object.reduce": "^1.0.0",
-    "undertaker-registry": "^1.0.0",
-    "fast-levenshtein": "^1.0.0"
+    "bach": "^1.2.0",
```

Comment:
```suggestion
    "bach": "^2.0.0",
```
This has been released

Reply:
@phated 
By upgrading to `bach 2.0.0`, many test cases failed. The cause is that we modified [this line](https://github.com/gulpjs/bach/blob/master/lib/helpers.js#L60) from `arr-flatten` because we presume the arguments of [`buildParallel`](https://github.com/gulpjs/bach/blob/master/lib/parallel.js#L12), etc. only flat arguments like `(taskWrapper1, taskWrapper2, ..., extensions)`.
However, `undertaker` passes arguments `([taskWrapper1, taskWrapper2, ...], extensions)` to their `bach`'s functions, [such as here](https://github.com/sttk/undertaker/blob/chore-normalize-repository/lib/parallel.js#L15).

Should we modify `bach`? Or should we modify `create(args, extensions)` to `create.apply(null, args.concat(extensions)` in `undertaker`?

Reply:
@sttk I believe we made a mistake in bach, we used `args = Array.prototype.slice.apply(args);` which I think was a mistaken recommendation from me. We should be using `args = Array.prototype.concat.apply([], args)` which will flatten the array 1-level deep. I will make this fix and add regression tests to bach

Reply:
bach 2.0.1 is released

Reply:
```suggestion
    "bach": "^2.0.1",
```